types/react/index.d.ts

```diff
@@ -1527,6 +1527,10 @@ declare namespace React {
             : P
             : P;
 
+    /**
+     * Merges the Prop types with an optional children prop
```

Comment:
This is just repeating the name. We don't need JSDoc here. The description should provide additional information if helpful.